src/styles/components/grid-square.scss

```diff
@@ -0,0 +1,29 @@
+.grid-square {
+  display: inline-flex;
+  flex-direction: row;
+  margin: 2rem;
+}
+
+.grid-square__input[type="checkbox"] {
```

Comment:
why do we need the checkbox type? did you consider a --checkbox if you really need to check that specifically?

Reply:
can you explain? i don't understand your question

Reply:
see reply below